extras/signers.json

```diff
@@ -45,13 +45,15 @@
     "davgarai": "0xAc1aA53108712d7f38093A67d380aD54B562a650"
   },
   "ozDefender": {
-    "checkpointer": "0xe9735f7D85a57BFb860c1E2c1c7B4F587BA0F6e7"
+    "checkpointer": "0xe9735f7D85a57BFb860c1E2c1c7B4F587BA0F6e7",
```

Comment:
```suggestion
    "checkpointerV1_old": "0xe9735f7D85a57BFb860c1E2c1c7B4F587BA0F6e7",
```

Reply:
Does this have any implications down the road if we rename the old checkpointer?

Then I would suggest let's call them
old_relayer_v1
relayer_v2